### PR TITLE
Disable old job group editor admin functionalities in favor of YAML

### PR DESCRIPTION
**Pull request by okurz**
This removes further code for the old job templates editor functionality
with a notice on the web page to save as YAML instead. The read-only
functionality is still included as non-admin users can not change the
job group to YAML.

Related progress issue: https://progress.opensuse.org/issues/44360
Fixes: [poo#60014](https://progress.opensuse.org/issues/60014)

### CODE REVIEW

**Comment by okurz** [comment]
Follow-up to https://github.com/os-autoinst/openQA/pull/2518 . We can also delay merging this for longer if we require. What should we do as next step, force any admin onto the save as YAML page?

**Comment by Martchus** [comment]
Likely you need to adapt tests (or throw them out).

File: templates/admin/job_template/index.html.ep [hunk 1]
```diff
@@ -84,9 +84,7 @@
                 <p id="loading"><i class="fa fa-spinner fa-spin"></i> Loading…</p>
 
                 <select id="machines-template" multiple="true"
```
Comment (Martchus) [inline templates/admin/job_template/index.html.ep]: It would be nice to make this one line (as well as the other occurrence).

Comment (okurz) [reply templates/admin/job_template/index.html.ep]: done

**Pull request merged**